.ci.yaml

```diff
@@ -65,7 +65,7 @@ platform_properties:
         []
       os: Mac-12
       device_type: none
-      cpu: x86
+      cpu: arm64
```

Comment:
First, attempting to run these tests only on arm64 to confirm they all pass on that arch.

Reply:
(This change will not be checked in, just confirming on this PR that all tasks pass on arm64 as well as x64).